codersdk/provisionerdaemons.go

```diff
@@ -265,3 +265,71 @@ func (c *Client) ServeProvisionerDaemon(ctx context.Context, req ServeProvisione
 	}
 	return proto.NewDRPCProvisionerDaemonClient(drpc.MultiplexedConn(session)), nil
 }
+
+type ProvisionerKey struct {
```

Comment:
Can we drop a comment that the hashed secret is intentionally omitted and should not be included in any http response?